call.go

```diff
@@ -181,13 +181,14 @@ func (c *calledFrom) succs(b *ssa.BasicBlock) bool {
 		return false
 	}
 
+	var called bool
 	for _, s := range b.Succs {
-		if !c.instrs(s.Instrs) && !c.succs(s) {
-			return false
+		if c.instrs(s.Instrs) || c.succs(s) {
+			called = true
```

Comment:
Why does not return immediately?

Reply:
In a defer function, the index of instr won't be 1.
So the loop will be returned before the `Instruction` was checked.
For an accurate validation, an iteration for "all of" `s.Succs` values is needed.

Reply:
I think this code only checks whether  the function was called in any succsesors yet.
Because there is an assignment which changes value of var called, `called = true`.